analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -149,7 +149,7 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
         fun parseLicenses(mavenProject: MavenProject) =
             mavenProject.licenses.mapNotNull { license ->
                 license.name ?: license.url ?: license.comments
-            }.toSortedSet()
+            }.filter { it.isNotBlank() }.toSortedSet()
```

Comment:
Maybe instead:

    (license.name ?: license.url ?: license.comments).takeUnless { it.isBlank() }

Reply:
@sschuberth I saw your comment too late, but it made me aware of another issue I fixed in #6000.